### PR TITLE
feat: on error put app in offline mode

### PR DESCRIPTION
**Pull request by cbernat**
### Add maintenance screen

https://cdn.fromdoppler.com/doppler-webapp/development-build2587/#/offline
Using GetUserData call, that we always do to get user state, if it were non reachable by a 404 or 503 http status code, that is what the load balances gives for a non available resource, put the app offline. 

ezgif com-video-to-gif (1)

image

image

@swidzinskiMS @fgchaio fyi

Pending:

- [x] Add new images
- [x] Add new styles
- [x] Add new text
- [x] Remove chat

image

### CODE REVIEW

File: src/services/app-session.ts [hunk 1]
```diff
@@ -27,6 +27,7 @@ interface AuthenticatedAppSessionWithDatahub
 export type AppSession =
   | { status: 'unknown' }
   | { status: 'non-authenticated' }
+  | { status: 'maintenance' }
```
Comment (andresmoschini) [inline src/services/app-session.ts:30]: Good idea!

File: src/services/session-manager.ts [hunk 2]
```diff
@@ -71,7 +71,11 @@ export class OnlineSessionManager implements SessionManager {
         } as AppSession, // Cast required because TS cannot resolve datahubCustomerId complexity
       );
     } catch (error) {
-      this.updateSession({ status: 'non-authenticated' });
+      if (!!error.response && (error.response.status === 503 || error.response.status === 404)) {
```
Comment (lucianaDilizio) [inline src/services/session-manager.ts:74]: Maybe we can use something like: 
`if ( error?.response.status === 503 || error?.response.status === 404)`
I'm not sure if this works fine.

Comment (cbernat) [reply src/services/session-manager.ts:74]: I've tried `error.response?.status` without success 😕

Comment (lucianaDilizio) [reply src/services/session-manager.ts:74]: Yes, it seems to be that is in that way.

Comment (cbernat) [reply src/services/session-manager.ts:74]: image
🤔

Comment (lucianaDilizio) [reply src/services/session-manager.ts:74]: 🤷‍♀️

**Pull request merged**